test/test_client_async.py

```diff
@@ -1,6 +1,8 @@
+import asyncio
+import ctypes
 import logging
 import time
-import unittest
+import pytest
```

Comment:
I guess this is causing mypy to crash. I don't know how this new test stuff works, but it looks like an error with the scope (pytest can't be found, because it is in the pytest file in venv??)